react/src/utils/endpoints.js

```diff
@@ -0,0 +1,17 @@
+function getBasePath() {
+  return 'https://slackclone001.herokuapp.com'
+}
+
+const ORGANIZATION_ID = '1ae5241e-a51b-11ec-b909-0242ac120002'
+
+export const USERS_LIST = `${getBasePath()}/users`
+
+export const CHANNEL_LIST = `${getBasePath()}/channel`
+
+export const MESSAGES_LIST = `${getBasePath()}/messages/4f0300c2-a51b-11ec-b909-0242ac120002/1ae5241e-a51b-11ec-b909/messages/4f0300c2-a51b-11ec-b909-0242ac120002/${ORGANIZATION_ID}`
```

Comment:
This is the wrong endpoint checkout https://github.com/web-slate/slack-app-clone-api/issues/2

@curioustushar